openedx/core/djangoapps/models/course_details.py

```diff
@@ -337,6 +343,17 @@ def recompose_video_tag(video_key):
             )
         return result
 
+    @staticmethod
+    def check_json_string(json_string):
+        """
+        Returns string JSON if json_string is valid json. Return None in another case.
```

Comment:
typo s/Returns/Return/ and s/string JSON/JSON string/ and s/valid json/valid JSON/ and s/in another case/in other cases

Reply:
Done